README.md

```diff
@@ -82,7 +82,7 @@ Install this project with:
 
```

Comment:
Missing key features and future features from readme, please all of the them are mandatory except optional faq.